### PR TITLE
remove styles page

### LINKED ISSUE #5844: Clarify where to place the dbt styles doc
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

@emmyoop shared this page: https://docs.getdbt.com/styles she found when googling it. We should find a home for it or put it in the docs github contributing folder. 

internal slack thread; https://dbt-labs.slack.com/archives/C05FWBP9X1U/p1721897622948869

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/styles

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
this page: https://docs.getdbt.com/styles can be found when googling 'styles'. 

this pr finds a permanent home fort he content in teh 'contributing' section of the repo.

Resolves #5844

### CODE REVIEW

File: website/vercel.json [hunk 1]
```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/src/pages/styles.js",
+      "destination": "https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/adding-page-components.md",
+      "permanent": true
```
Comment (JKarlavige) [inline website/vercel.json]: It looks like the redirect isn't being picked up here: https://docs-getdbt-com-git-remove-style-dbt-labs.vercel.app/styles

The `source` here just needs to match the live url of the page, which would be `/styles` in this case!

**Review by JKarlavige — CHANGES_REQUESTED** [review]
Hey @mirnawong1, just requesting one change to get the redirect working. Once adjusted this should be good to 🚢 !

File: website/vercel.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/src/pages/styles.js",
```
Comment (mirnawong1) [inline website/vercel.json]:
```suggestion
      "source": "/styles",
```

File: website/vercel.json (hunk 1 above)
Comment (mirnawong1) [reply website/vercel.json]: thanks so much for reviewing @JKarlavige ! I've updated this on my end and ready for re review whenever you can 🙏

**Review by JKarlavige — APPROVED** [review]
lgtm! verified the redirect is firing correctly when hitting https://docs-getdbt-com-git-remove-style-dbt-labs.vercel.app/styles 

:shipit:

**Comment by mirnawong1** [comment]
you're the best, thank you so much for the quick review @JKarlavige !

**Pull request merged**